app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java

```diff
@@ -1700,10 +1700,14 @@ private void showsnackbar(Boolean result) {
         }
     }
 
-    private void checkNoSearchResults(String result){
-        textView.setText(getString(R.string.null_search_result) + " " + '"' + result + '"' );
-        textView.setTextColor(getTextColor());
-        textView.setVisibility(View.VISIBLE);
+    private void checkNoSearchResults(String result,int length){
+        if (length>0) {
+            textView.setText(getString(R.string.null_search_result) + " " + '"' + result + '"');
+            textView.setTextColor(getTextColor());
```

Comment:
Just hide the textview

Reply:
So basically you want to remove the textview from screen?

Reply:
or you are asking me to implement View.Gone?

Reply:
View.gone